extensions/stackdriver/stackdriver.cc

```diff
@@ -350,7 +350,7 @@ void StackdriverRootContext::record() {
   ::Extensions::Stackdriver::Metric::record(
       outbound, local_node, peer_node, request_info,
       !config_.disable_http_size_metrics());
-  if (enableServerAccessLog() && shouldLogThisRequest()) {
+  if (enableServerAccessLog() && shouldLogThisRequest(request_info)) {
```

Comment:
With this code, Isn't it that all entries exported will just have `filtered: true` label? So I am confused on the meaning of this label? Could you add some comment about this?

Reply:
Only the logs that were sampled will have `filtered: true` label

Reply:
Yeah, but every log entry reaches the backend has `filtered: true` label, doesn't it? If this label presents on all log entries with the same value, it essentially brings no value, no?

Reply:
As talked offline... this will only mark the label "filtered":"true" changed to "log_sampled":"true" for whom kAccessLogPolicyKey is set. Which is set by Access Log Policy sampling filter when places in front of stackdriver.